.github/workflows/build.yml

```diff
@@ -12,3 +12,7 @@ jobs:
       run: msbuild RTCV.sln /t:restore
     - name: Build with MSBuild
       run: msbuild RTCV.sln
+    - name: Run tests
+      run: dotnet test
+    - name: Codecov
+      uses: codecov/codecov-action@v1.0.13
```

Comment:
I created `coverage.yml` and added the CodeCov action there.

I did this so that coverage wouldn't be reported for every pull request (I'm not sure how Codecov handles coverage reports for PRs)

Reply:
If we do end up wanting coverage reports on PRs we can merge `build.yml` and `coverage.yml`

Reply:
If I'm not mistaken, it'll append what kind of coverage comes with the PR to the PR itself.

Reply:
Ooo, in that case, let me delete `coverage.yml` and fold that configuration into `build.yml`

Reply:
Agreed. And done.